src/ontology/ro-edit.owl

```diff
@@ -5988,6 +5990,25 @@ AnnotationAssertion(rdfs:label obo:RO_0009005 "has primary substance added"@en)
 SubObjectPropertyOf(obo:RO_0009005 obo:RO_0009001)
 IrreflexiveObjectProperty(obo:RO_0009005)
 
+# Object Property: obo:RO_0009006 (assay measures characteristic)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0009006 "A relation between an assay and a characteristic, in which the assay generates a data item which is a measure of a characteristic."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:RO_0009006 <https://orcid.org/0000-0002-8844-9165>)
+AnnotationAssertion(terms:date obo:RO_0009006 "2023-05-23T15:24:15Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0009006 "assay measures characteristic"@en)
+InverseObjectProperties(obo:RO_0009006 obo:RO_0009007)
+ObjectPropertyDomain(obo:RO_0009006 obo:OBI_0000070)
+ObjectPropertyRange(obo:RO_0009006 obo:BFO_0000020)
+
+# Object Property: obo:RO_0009007 (characteristic measured by assay)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0009007 "Inverse of 'measures characteristic'"@en)
+AnnotationAssertion(obo:IAO_0000117 obo:RO_0009007 <https://orcid.org/0000-0002-8844-9165>)
```

Comment:
Could you please change the annotation to `terms:contributor`?

Reply:
done!

Reply:
Thanks!